sound/soc/intel/boards/sof_sdw.c

```diff
@@ -249,6 +249,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_BT_OFFLOAD_SSP(2) |
 					SOF_SSP_BT_OFFLOAD_PRESENT),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc"),
+			DMI_EXACT_MATCH(DMI_PRODUCT_SKU, "0AF3"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
+					SOF_RT715_DAI_ID_FIX |
+					SOF_SDW_FOUR_SPK),
```

Comment:
you may want to add a comment that there's no jack here, to avoid confusion on the use of JD1.